src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml

```diff
@@ -141,8 +143,8 @@
                     <div class="legal_text">
                         <p>
                             © 2019 - DevAdventCalendar<br />
-                            <a href="~/polityka.pdf" target="_blank">Polityka prywatności</a>
-                            <a href="~/regulamin.pdf" target="_blank">Regulamin</a>
+                            <a href="~/polityka.pdf" target="_blank">@PrivacyPolicy</a>
+                            <a href="~/regulamin.pdf" target="_blank">@Policy</a>

```

Comment:
use `TermsOfUse`

Reply:
There are similars words so I changed to:

PrivacyPolicy -> Polityka prywatności
TermsOfUse -> Regulamin
AcceptTermsOfUse ->  Zostawiając tutaj swojego maila zgadzasz się na postanowienia regulaminu dostępnego tutaj